pkg/sentry/fsimpl/fuse/fusefs.go

```diff
@@ -227,3 +227,81 @@ func (i *Inode) Open(ctx context.Context, rp *vfs.ResolvingPath, vfsd *vfs.Dentr
 	}
 	return fd.VFSFileDescription(), nil
 }
+
+// Stat implements kernfs.Inode.Stat.
+func (i *Inode) Stat(ctx context.Context, fs *vfs.Filesystem, opts vfs.StatOptions) (linux.Statx, error) {
+	conn := fs.Impl().(*filesystem).conn
```

Comment:
Maybe the inode should have a reference to the filesystem to make this nicer?

Reply:
Yeah it could be. However `fs` will still be passed in as an argument because other fs implementation has no reference in its inode struct. To be consistent with others, I think it's fine so far.